project/src/pages/main/main.tsx

```diff
@@ -4,13 +4,23 @@ import Sort from '../../components/sort/sort';
 import PlacesList from '../../components/places-list/places-list';
 import Map from '../../components/map/map';
 import {Offers} from '../../types/offers';
+import {useState} from 'react';
 
 type MainProps = {
   placesCount: number;
   offers: Offers;
 }
 
 function Main({placesCount, offers}: MainProps): JSX.Element {
+  const setActiveOffer = useState(0)[1];
```

Comment:
Надя разве в демо проекте используют такую запись?
```suggestion
  const [, setActiveOffer] = useState(0);
```